tools/rcverify.sh

```diff
@@ -67,14 +64,11 @@ TGZ=$NAME-$V-sources.tar.gz
 # this is a constructed name for the keys file
 KEYS=$RC-$V-KEYS
 
-NOTICE=$(cat << END
-Apache $DESCRIPTION
-Copyright 2016-2021 The Apache Software Foundation
+NOTICE_REGEX='^Apache .+
```

Comment:
This is a positive change - thanks for improving the checks.
Removing the component description from the check could lead to an error where someone copies the notice file from one repo and forgets to update the name. Will have to be diligent about this.

It's nice to have one fewer argument to rcverify.